python/ray/util/client/server/proxier.py

```diff
@@ -221,6 +228,10 @@ def _check_processes(self):
         while True:
             with self.server_lock:
                 for client_id, specific_server in list(self.servers.items()):
+                    try:
+                        specific_server.wait_ready(0.1)
+                    except futures.TimeoutError:
+                        continue
                     poll_result = specific_server.process_handle(
                     ).process.poll()
                     if poll_result is not None:
```

Comment:
At https://github.com/ray-project/ray/blob/39cdca351adbea4310531721c0613b3b429bb242/python/ray/util/client/server/proxier.py#L180 this is treated as an error. Do we need to log error here as well?

Reply:
Nope, this is okay if it times out because the cleanup thread runs constantly. The time out here just means that the server has not been created yet!